### DISCUSSION TITLE
How do I join the community

### DISCUSSION THREAD

**Discussion by Dev-Yusuf** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#48] [Outreachy 2022] [answered]
Hi everyone. I'm an outreachy applicant, I very interested in this project. I tried joining the community through the provided link but I was shown the form has reached it maximum quota. Is there anything I can do?

**Comment by BatoolMM** [comment, marked as answer]
Hi @Dev-Yusuf, this is [the form](https://form.jotform.com/222902545384356) - let me know if you still struggle.
